whirlpool/eventsocket.py

```diff
@@ -133,7 +133,7 @@ async def _run(self):
                             connected_msg_done = True
                             continue
 
-                        if not subscribe_msg_done:
```

Comment:
I forgot about this one - found it's not waiting for the connected_msg_done before sending the subscribe_msg_done.

Reply:
this elif is not needed, since the if block has `continue` already

Reply:
True - I thought I had traced this through and it was behaving differently, but on second review it was working as expected.